src/templates/company/index.js

```diff
@@ -80,13 +80,17 @@ export default {
     const { domain, tokens, voting } = data
     const { tokenName, tokenSymbol, members } = tokens
     const baseStake = new BN(10).pow(new BN(18))
-    const stakes = members.map(([_, stake]) => baseStake.muln(stake).toString())
+    const stakes = members.map(([_, stake]) =>
+      baseStake.mul(new BN(stake)).toString()
```

Comment:
@facuspagnuolo We should make sure to pass `stake` back from the Tokens page as a string (`new BN()` will fail if the number is too big).